benchmarks/2019.json

```diff
@@ -4329,8 +4293,8 @@
       0,
```

Comment:
@lamroger - just got word from policy we need to remove 425 registry benchmark too. can we delete here please?